test/prettyprinting.jl

```diff
@@ -0,0 +1,6 @@
+using Dates, DimensionalData
+using DimensionalData: Time, X
+timespan = DateTime(2001):Month(1):DateTime(2001,12)
+t = Time(timespan)
+x = X(Vector(10:10:500))
+A = DimensionalArray(rand(12,length(x)), (t, x))

```

Comment:
Can we run `show` on these? We probably don't need an actual test, but it would be good to [run the code](https://codecov.io/gh/rafaqz/DimensionalData.jl/pull/33/diff?src=pr&el=tree#diff-c3JjL3ByZXR0eXByaW50Lmps) to make sure it doesn't error.